Formula/pulumi.rb

```diff
@@ -0,0 +1,30 @@
+class Pulumi < Formula
+  desc "Define cloud applications and infrastructure in your favorite language"
```

Comment:
I think this sounds a little... sales pitchy. The description on GitHub is maybe worth considering instead:
```
Cloud native development platform
```
or:
```
Cloud native language-agnostic development platform
```
or something similar.